workshops/erc20-mts/scripts/cli-keys.sh

```
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+source .local.env
+
+injectived keys $* --keyring-backend=test
```

Shellcheck (0.10.0)

[warning] Use "$@" (with quotes) to prevent whitespace problems.

(SC2048)